js/main.js

```diff
@@ -0,0 +1,28 @@
+const $photoUrl = document.querySelector('#photo-url');
+const $img = document.querySelector('img');
+
+$photoUrl.addEventListener('input', e => {
+  $img.setAttribute('src', e.target.value);
+});
+
+const $form = document.querySelector('#entry-form');
+
+// console.log($form.elements);
```

Comment:
Remove all `console.log` before submitting! By creating a pull request, you are essentially saying this is the best I can do, and leaving old code or unused console logs shows that you aren't checking your work before submitting!